content/en/docs/installing-polo/index.md

```diff
@@ -3,31 +3,26 @@ title: Installing PoLo
 description: How to install PoLo on your device.
 weight: 10
 ---
-
 PoLo is currently available for Android and iOS devices.
 
-## Android
-
+### Android
 PoLo is available in the Google Play Store. You can install it by clicking the badge below:
 
-<a href='https://play.google.com/store/apps/details?id=com.ham2k.polo.beta'  target='_blank'><img src='/google-play-badge.png' alt='Google Play Store' width='240' /></a>
-
-The specification of some Android devices is such that a Lite version of PoLo has been produced. More information on this version and how to install it is found on the [Installing PoLo Lite page](./installing-pololite/).
+<a href='https://play.google.com/store/apps/details?id=com.ham2k.polo.beta'  target='_blank'><img src='google-play-badge.png' alt='Google Play Store' width='186' /></a>
 
-## Apple iOS
+The specification of some Android devices is such that a Lite version of PoLo has been produced. More information on this version and how to install it is found on the [PoLo Lite page](../pololite/).
```

Comment:
Some Android devices, notably some Samsung and Lenovo tablets, might not have enough free RAM to run the regular version of PoLo and might crash on startup. We have produced a **PoLo Lite** version to work around this problem. More information on this version and how to install it is found on the [PoLo Lite page](../pololite/).